androidProjects.yml

```diff
@@ -0,0 +1,10 @@
+name: androidProjects
+desc: this repository consists of some basic projects of android studio
```

Comment:
Suggested change: `A collection of basic Android projects that are tailored to new contributors`